### ISSUE TITLE
make reference number a bit more future proof

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-meinberlin#4171]
**URL:** 
**user:** 
**expected behaviour:** 
**behaviour:** At some point all digits of the number will be used. We should make (display of) the number longer well before!
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 This is for the ideas and plans, right? As the number starts with the year, we would need to have more than 99.999 Ideas or plans in one year to have a problem, no? we are far from there yet.

**Comment by fuzzylogic2000** [comment]
@CarolingerSeilchenspringer No, we never start again, the second part is the ID, so it is always counting up, we don't care about the year. The year is just there to make it look more serious.
That means whenever we have more than 99.999 items we already have a problem. We have about 80.000 left, which is still a lot. But with more and more people using mB, we should be early enough changing that to 6 digit.

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 ok, then: issue or story?
